### PR TITLE
refactor: Remove unused stuffs

### PR DESCRIPTION
**Pull request by sanjivkrish**
## Description

- Bootstrapped from zenjob/service-template-nodejs
- Removing unused stuff like Kafka, Zookeeper, TypeORM, migration scripts
- Infra setup

### CODE REVIEW

File: infra/jenkins/Jenkinsfile-live [hunk 1]
```diff
@@ -8,7 +8,6 @@ String teamSlackChannel = "platform-public"
 String jenkinsBuildChannel = "jenkins-ci"
 String jenkinsPrChannel = "jenkins-ci"
```
Comment (vaidehi27) [inline infra/jenkins/Jenkinsfile-live:9]: need to change to these configs I guess

File: .eslintrc [hunk 2]
```diff
@@ -1,68 +1,61 @@
...
-    "parser": "@typescript-eslint/parser",
-    "plugins": [
-        "prettier",
-        "import",
-        "@typescript-eslint"
+  "parser": "@typescript-eslint/parser",
+  "plugins": ["prettier", "import", "@typescript-eslint"],
+  "extends": [
+    "eslint:recommended",
+    "airbnb-base",
```
Comment (vaidehi27) [inline .eslintrc:6]: I think we can make this file simple as in zj-unleash-react-client lib

**Comment by sanjivkrish** [comment]
> `tslint.json` file can be removed

This file comes with the default template (used for linting currently) and not the focus for this PR.
This PR is just to remove the unused stuff(and replace template-nodejs name wherever used) and @tcondeixa told he will add the infra part as part of this PR (Will update the description once it is done).

This is not the final version of the app. For the rest of the stuff, we can create subsequent PRs to keep track. If you think anything that comes with the default-template that can be removed or renamed from 'template-nodejs' please feel free to suggest as part of this PR.

File: .eslintrc (hunk 2 above)
Comment (sanjivkrish) [reply .eslintrc:6]: Sure. But not part of this PR, I will create another PR to replace the lint (from tslint to eslint).

**Pull request merged**